### PR TITLE
79 feature/micro service/master

### PR DESCRIPTION
**Pull request by ji2myji2my**

### CODE REVIEW

File: Game/db.sqlite3
Comment (DoubleV37) [inline Game/db.sqlite3:1]: Why there is this file ?

File: docker-compose.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  guni_game:
+    container_name: guni_game
+    build: Services/Gunicorn_Game
     restart: always
     env_file: .env
     volumes:
-      - django_data:/app/Project
+      - Game:/app/Project
     depends_on:
       postgres:
```
Comment (DoubleV37) [inline docker-compose.yml:10]: need change to game_db instead of database

**Review by DoubleV37 — CHANGES_REQUESTED** [review]
Need to add rule in gitignore file to ignore the staticfiles folder in Game django project.
The User app in Game project is not in the apps folder, not in the settings.py and has no urls.py, why ?

**Comment by ji2myji2my** [comment]
> Need to add rule in gitignore file to ignore the staticfiles folder in Game django project. The User app in Game project is not in the apps folder, not in the settings.py and has no urls.py, why ?

Done ✔️

File: docker-compose.yml (hunk 1 above)
Comment (ji2myji2my) [reply docker-compose.yml:10]: > need change to game_db instead of database

I don't understand what i have to do here.

**Comment by ji2myji2my** [comment]
> The User app in Game project is not in the apps folder, not in the settings.py and has no urls.py, why ?

My mistake. I'm correcting it immediately. It was during a development session that I created this but yes the right place for the user folder is in apps.

File: docker-compose.yml (hunk 1 above)
Comment (DoubleV37) [reply docker-compose.yml:10]: The depends on is not on the right database container, for game is game_db or something like that.

**Comment by ji2myji2my** [comment]
> > The User app in Game project is not in the apps folder, not in the settings.py and has no urls.py, why ?
> 
> My mistake. I'm correcting it immediately. It was during a development session that I created this but yes the right place for the user folder is in apps.

Moving: User in apps and adjusting settings.py Done ✔️

missing handling of urls.py can you give me more details.

**Pull request closed without merging**